scripts/find-broken-links

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
This is nice, great work.

Reply:
@weblancaster If you like it, put an approve on it 8-)